mainwindow.cpp

```diff
@@ -871,15 +851,15 @@ void MainWindow::showJohnFinished()
     checkNToggleActionsLastSession();
     // When John stops we need to stop timer and to look status last
     // time.
-    // TODO: currently if john crashed we do not call john.
+    // Currently if john crashed we do not call john.
```

Comment:
I verified and showJohnFinished slot is called when we sent SEGV signal so I don't know what this comment meant.